snuba/api.py

```diff
@@ -162,16 +162,80 @@ def query(validated_body=None, timer=None):
     )
 
 
+def format_query(dataset, body, table, prewhere_conditions, final) -> str:
+    """Generate a SQL string from the parameters."""
+    aggregate_exprs = [util.column_expr(dataset, col, body, alias, agg) for (agg, col, alias) in body['aggregations']]
```

Comment:
@markstory  you have a PR that changes this. You probably want to keep an eye on this one.